scm/driver/gitea/git.go

```diff
@@ -24,7 +25,11 @@ func (s *gitService) FindBranch(ctx context.Context, repo, name string) (*scm.Re
 }
 
 func (s *gitService) FindCommit(ctx context.Context, repo, ref string) (*scm.Commit, *scm.Response, error) {
-	path := fmt.Sprintf("api/v1/repos/%s/git/commits/%s", repo, ref)
+	ref = strings.TrimPrefix(ref, "refs/")
```

Comment:
I noticed this code trims `refs/`.  Is it correct that `refs/heads/master` should be sent to the Gitea API as `heads/master`?  I just wanted to confirm ...

Reply:
With the latest gitea image it was necessary yes. This is the new API call [go-gitea/gitea#10915](https://github.com/go-gitea/gitea/pull/10915). It would of course be nice if someone else could verify it also works for him/her.

Reply:
Hello @bradrydzewski , we spoke about this issue (failing cron builds on newer Gitea) in Drone's Gitter room last Friday, could I ask what is the current status? :)

Reply:
`refs/heads/master` works fine, trimming is unnecessary.